cmd/main.go

```diff
@@ -105,7 +106,7 @@ func main() {
 	bindEnv(logger, "verbose_debugging", "VERBOSE_DEBUGGING")
 	// Set default values
 	viper.SetDefault("cluster_creds", "clustercreds")
-	viper.SetDefault("cilium_namespace", "kube-system")
+	viper.SetDefault("cilium_namespaces", "kube-system,gke-managed-dpv2-observability")
```

Comment:
Viper natively supports string slices.
`BindEnv` is supposed to parse env variable values as comma-separated in this case, but that needs to be tested.
Can you please test this?
```suggestion
	viper.SetDefault("cilium_namespaces", []string{"kube-system", "gke-managed-dpv2-observability"})
```

Make sure to test by setting `CILIUM_NAMESPACES` to e.g. `"foo,bar"` and see that it gets parsed into `[]string{"foo", "bar"}`.